### DISCUSSION TITLE
Product feedback category for "Projects"

### DISCUSSION THREAD

**Discussion by uyriq** [community/community#30208] [Projects and Issues]
Didn't find a **project** category in this community, although other github products are represented here, so maybe my question has a right to be asked. I have been using github projects for a while now and have a question about sending issues from the command line using gh (github cli tool). the answer to the --project flag "my project name" always is `could not add to project: 'my project name' not found`. a similar question was already here https://github.com/cli/cli/issues/4547 maybe no solution yet, but I do not know about it.

**Comment by queenofcorgis** [comment]
The Issues category does encompass Projects as well, thanks for posting in the proper category 😄

**Reply by queenofcorgis** [reply]
I've added a Projects label to this so it is seen by the proper folks.
